examples/f5_bigip/README.md

```diff
@@ -0,0 +1,281 @@
+# Load balancing a set of servers using F5
```

Comment:
This title should reflect what we are doing here:
**Configuring SSL termination with _Venafi Ansible Role_ on a set of HTTP servers that are load balanced by F5-BIG-IP.**